Podcast/User.swift

```diff
@@ -9,38 +9,25 @@
 import UIKit 
 import SwiftyJSON
 
-class User: NSObject {
+class User: SearchUser {
     
-    var id: String
-    var firstName: String
-    var lastName: String
-    var username: String
-    var numberOfFollowers: Int
     var numberOfFollowing: Int
     var favoriteEpisodes: [Episode]?
     var subscriptions: [Series]?
     var following: [User]
```

Comment:
do we ever get following returned? In the model or favoriteEpisodes / subscriptions? idk if we should store this is a user model tbh

Reply:
I don't think so, but it might be useful for when we cache things later on.

Reply:
let's delete it for now and add it back in when we need to .. I don't want us assuming we have them for a user